server.py

```diff
@@ -0,0 +1,20 @@
+import json
```

Comment:
Faraz I shared PEP8 style guide with you. Did you read it thoroughly? Import statements should always be followed by new lines.

Reply:
`my_server_1.py` is a childish name for a file. Please use descriptive names. Call it `server.py`. We don't need this current `server.py` going forward.